### PR TITLE
first batch of Readme updates

### PR DESCRIPTION
**Pull request by xinity**
README update WIP.

leave the PR open until all updates are committed.
comments welcome of course :)

### CODE REVIEW

File: README.md
Comment (tboerger) [inline README.md]: I would not mention that here, we can skip the maintainance of this part. You can see that anyway on the people page of the org.

File: README.md
Comment (tboerger) [inline README.md]: `You must read the [contributors guide]() before starting to work on a pull request`

File: README.md
Comment (tboerger) [inline README.md]: This should not be a headline, beside that this branch will be 1.0.0

**Review by tboerger — CHANGES_REQUESTED** [review]

File: README.md
Comment (xinity) [reply README.md]: agree @lunny 
i though at first to remove this line and simply link to the release page.

**Comment by xinity** [comment]
I've taken care of changes requested, but github stills ask for them to be taken care of.
Did i miss something ?

**Comment by strk** [comment]
I can see a "Dismiss review" link next to the "requested changes" record. I guess all maintainers do see that.
But given @tboerger requested the change I guess he's the one who should click that link, to confirm his request was addressed

File: README.md
Comment (strk) [inline README.md]: I think versions have 3 numbers, so would be 1.0.0 (but there's no such release made yet).
I agree on dropping this paragraph completely and just link to the releases page to figure out what the "current version" is.

File: README.md
Comment (strk) [inline README.md]: @tboerger asked to use "contributors guide" here, IIRC

File: README.md
Comment (tboerger) [inline README.md]: security@gitea.io

Comment (strk) [inline README.md]: @lunny wasn't there a wider address to contact owners/maintainers ?

File: README.md
Comment (tboerger) [inline README.md]: Please add a blank line here like on every headline

File: README.md
Comment (tboerger) [inline README.md]: Gogs

File: README.md
Comment (strk) [inline README.md]: I would drop the gogs.io links, those would not be releases of Gitea anyway.
Maybe also drop the count of ways to install, as that number is going to change a few times before becoming stable :)

**Review by strk — CHANGES_REQUESTED** [review]

File: README.md
Comment (xinity) [reply README.md]: i agree but then there won't be any installation documentation links until we write one.
Doesn't matter ?

File: README.md
Comment (strk) [inline README.md]: How about drop the version number (will easily get outdated) and just point to the releases page ?

File: README.md
Comment (strk) [inline README.md]: is using *Gitea ?

File: README.md
Comment (strk) [inline README.md]: Drop the count, those ways are going to change and we'll forget to update the count :)

File: README.md
Comment (strk) [inline README.md]: Capitalize See ?

**Comment by strk** [comment]
Reviewed, feel free to ignore my review and have my LGTM :)

File: README.md
Comment (xinity) [reply README.md]: not sure about this one

File: README.md
Comment (xinity) [reply README.md]: done will push

File: README.md
Comment (xinity) [reply README.md]: done will push

File: README.md
Comment (xinity) [reply README.md]: done will push

File: README.md
Comment (tboerger) [inline README.md:26]: I would keep that and link to the godocs.

File: README.md
Comment (tboerger) [inline README.md]: Numbering?

File: README.md
Comment (tboerger) [inline README.md]: I would drop that for now.

**Review by tboerger — CHANGES_REQUESTED** [review]

File: README.md
Comment (ghost) [inline README.md]: Possibly `[Contributors]`, `[Maintainer]` and `[Translators]`?

File: README.md
Comment (ghost) [reply README.md]: Agreed, we just became open source after all.

**Review by strk — APPROVED** [review]

File: README.md
Comment (xinity) [reply README.md]: done will push

File: README.md
Comment (xinity) [reply README.md]: done will push

File: README.md
Comment (xinity) [reply README.md:26]: done will push

File: README.md
Comment (strk) [inline README.md:26]: Gogs came back ?

**Comment by xinity** [comment]
@strk are you sure?

File: README.md
Comment (strk) [reply README.md:26]: sorry, I was wrong, all looks fine

**Comment by lunny** [comment]
seems it's ready for merge. LGTM

File: README.md
Comment (bkcsoft) [inline README.md:63]: `fork of [Gogs]` sounds better no?

**Comment by strk** [comment]
On Wed, Nov 09, 2016 at 06:30:13AM -0800, bkcsoft wrote:

> `fork of [Gogs]` sounds better no?

Nitpicking

**Comment by bkcsoft** [comment]
@strk which is why I left it as a comment instead of `required change` :wink:

**Comment by strk** [comment]
Do do we have your LGTM @bkcsoft ? So we can merge and move on

**Comment by lunny** [comment]
@tboerger we need your approval changes.

**Review by tboerger — APPROVED** [review]

**Pull request merged**
